windows-installer-agent.ps1

```diff
@@ -0,0 +1,148 @@
+# Viam Agent Installation Script for Windows
+[CmdletBinding()]
+param(
+    [switch]$Silent = $false
+)
+
+$ErrorActionPreference = 'Stop'
+
+# Check for admin privileges
+$isAdmin = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+if (-not $isAdmin) {
+    if (-not $Silent) {
+        Write-Host "This script requires administrator privileges. Attempting to elevate..."
+    }
+    
+    # Self-elevate the script
+    $scriptPath = $MyInvocation.MyCommand.Path
+    Start-Process powershell.exe -ArgumentList "-ExecutionPolicy Bypass -File `"$scriptPath`" -Silent" -Verb RunAs
+    exit
+}
+
+if (-not $Silent) {
+    Write-Host "Starting Viam Agent installation..."
+}
+
+# Define installation paths
+$rootPath = "C:\opt\viam"
+$cachePath = Join-Path $rootPath "cache"
+$binPath = Join-Path $rootPath "bin"
+$agentCURLFileName = "viam-agent-stable-windows-x86_64"
```

Comment:
@ale7714 I updated this and the URL below to the stable URL, but have not tested -- kicking this back to you